addons/account/models/account_payment.py

```diff
@@ -638,32 +684,25 @@ def _synchronize_from_moves(self, changed_fields):
                 all_lines = move.line_ids
                 liquidity_lines, counterpart_lines, writeoff_lines = pay._seek_for_lines()
 
-                if len(liquidity_lines) != 1 or len(counterpart_lines) != 1:
-                    raise UserError(_(
-                        "The journal entry %s reached an invalid state relative to its payment.\n"
-                        "To be consistent, the journal entry must always contains:\n"
-                        "- one journal item involving the outstanding payment/receipts account.\n"
-                        "- one journal item involving a receivable/payable account.\n"
-                        "- optional journal items, all sharing the same account.\n\n"
-                    ) % move.display_name)
+                if len(liquidity_lines) != 1:
+                    raise UserError(_("Journal Entry %s is not valid. In order to proceed the journal "
+                        "items must include one outstanding payments/receipts account.", move.display_name))
```

Comment:
You indentation is wrong too.
Either (really ugly and bad practice imo[1], but indentation checks out)
```python
myfunction_call(param1,
                param2)
```
Or
```python
myfunction_call(
    param1,
    param2,
)
```

This would by far IMHO be the best solution
```python
                raise UserError(_(
                    "Journal Entry %s is not valid. In order to proceed the journal items must"
                    " include one and only one outstanding payments/receipts account.",
                    move.display_name,
                ))
```

____________
[1] because the if you change the name of the function, you have to change every line of argument, and because it leaves less space on the line.

Reply:
In order to proceed, the journal items must ... > A comma after proceed will make it even clearer.